log2ram

```diff
@@ -36,28 +36,28 @@ syncFromDisk () {
     if [ "$ZL2R" = true ]; then
         TP_SIZE=$LOG_DISK_SIZE
     fi
-    
-    if [ ! -z "$(du -sh -t "$TP_SIZE" $HDD_LOG/ | cut -f1)" ]; then
-        echo "ERROR: RAM disk for "$HDD_LOG/" too small. Can't sync."
-        umount -l $RAM_LOG/
-        umount -l $HDD_LOG/
+
+    if [ -n "$(du -sh -t "$TP_SIZE" "$HDD_LOG"/ | cut -f1)" ]; then
+        echo "ERROR: RAM disk for \"$HDD_LOG/\" too small. Can't sync."
+        umount -l "$RAM_LOG"/
+        umount -l "$HDD_LOG"/
         if [ "$MAIL" = true ]; then
-            echo "LOG2RAM : No place on RAM for "$HDD_LOG/" anymore, fallback on the disk" | mail -s 'Log2Ram Error' root;
+            echo "LOG2RAM : No place on RAM for \"$HDD_LOG/\" anymore, fallback on the disk" | mail -s 'Log2Ram Error' root;
         fi
         exit 1
     fi
 
     if [ -x "$(command -v rsync)" ]; then
-        rsync -aXv --inplace --no-whole-file --delete-after $HDD_LOG/ $RAM_LOG/ 2>&1 | tee -a $LOG2RAM_LOG
+        rsync -aXv --inplace --no-whole-file --delete-after "$HDD_LOG"/ "$RAM_LOG"/ 2>&1 | tee -a "$LOG2RAM_LOG"
     else
-        cp -rfup $HDD_LOG/ -T $RAM_LOG/ 2>&1 | tee -a $LOG2RAM_LOG
+        cp -rfup "$HDD_LOG"/ -T "$RAM_LOG"/ 2>&1 | tee -a "$LOG2RAM_LOG"
     fi
 
 }
 
 wait_for () {
-    while ! grep -qs "$1" /proc/mounts; do
-        sleep 0.1
+    while ! mount | grep "$1" > /dev/null; do
```

Comment:
Is there a specific reason not to simply use `findmnt` instead of grepping?

Reply:
Not realy, but is it safe to use? Is it available on all systems?

Reply:
It is part of `util-linux` and `man 8 mount` has this to say:

```
   Listing the mounts
       The listing mode is maintained for backward compatibility only.

       For more robust and customizable output use findmnt(8), especially in your scripts.  Note that control characters in the mountpoint name are replaced with '?'.

       The following command lists all mounted filesystems (of type type):

              mount [-l] [-t type]

       The option -l adds labels to this listing.  See below.
```

Reply:
OK, changed: https://github.com/azlux/log2ram/pull/130/commits/6e110ae11c84353ee608e68269328d98b699a700